### ISSUE TITLE
[ Accessibility WCAG 2.0/WCAG 2.1 ] Components violations

### ISSUE THREAD

**Issue by DlgSHi** [Sage/carbon#3028]
### Current behaviour

Components: 
- dialog-full-screen, 
- dialog, 
- flash, 
- pages,
- confirm,
- Animated menu button,
- Alert,
- Sidebar,
- Popover Container,
- Advanced Color Picker
have accessibility violations after changing its state to `open`.

### Expected behaviour

We should see no `errors` in cypress console (all tests should be marked as `pass`)
### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| carbon-react     | 31.1.1 |
| carbon-factory   |
| react-scripts    | 
| React            |
| Browser          | all |
| npm              | 6.0.0 |
| Operating System | Mac & CI (ubuntu) |

**Comment by jamime** [comment]
FE-2894
